### ISSUE TITLE
Poetry2nix broken with newer nixpkgs due to changes in the wheel derivation

### ISSUE THREAD

**Issue by DolceTriade** [nix-community/poetry2nix#1347]
### Describe the issue
https://github.com/NixOS/nixpkgs/commit/3cd71e0ae67cc48f1135e55bf78cb0d67b53ff86 updated the wheel derivation to have different input args.

Poetry2nix hard codes the old arguments: https://github.com/nix-community/poetry2nix/blob/master/overrides/default.nix#L2864-L2867

We need to have a version check and choose one or the other.
```
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'python-requirements'
         whose name attribute is located at /nix/store/g68f5abh3xhcz8xsdlfw7wkgkkcx3nwy-source/pkgs/stdenv/generic/make-derivation.nix:300:7

       … while evaluating attribute 'text' of derivation 'python-requirements'

         at /nix/store/g68f5abh3xhcz8xsdlfw7wkgkkcx3nwy-source/pkgs/build-support/trivial-builders/default.nix:148:16:

          147|     runCommand name
          148|       { inherit text executable checkPhase allowSubstitutes preferLocalBuild;
             |                ^
          149|         passAsFile = [ "text" ];

       error: function 'anonymous lambda' called with unexpected argument 'bootstrapped-pip'

       at /nix/store/g68f5abh3xhcz8xsdlfw7wkgkkcx3nwy-source/pkgs/development/python-modules/wheel/default.nix:1:1:

            1| { lib
             | ^
            2| , buildPythonPackage
```

**Comment by cpcloud** [comment]
This should be fixed by https://github.com/nix-community/poetry2nix/pull/1329. Can you give that a try?

**Comment by adisbladis** [comment]
Closed by #1329.
